warehouse/integrations/testhelper/setup.go

```diff
@@ -242,3 +250,166 @@ func EnhanceWithDefaultEnvs(t testing.TB) {
 		t.Setenv("LOG_LEVEL", "DEBUG")
 	}
 }
+
+func UploadLoadFile(
+	t testing.TB,
+	fm filemanager.FileManager,
+	fileName string,
+	tableName string,
+) filemanager.UploadedFile {
+	t.Helper()
+
+	f, err := os.Open(fileName)
+	require.NoError(t, err)
+	defer func() { _ = f.Close() }()
+
+	loadObjectFolder := "rudder-warehouse-load-objects"
+	sourceID := "test_source-id"
+
+	uploadOutput, err := fm.Upload(
+		context.Background(), f, loadObjectFolder,
+		tableName, sourceID, uuid.New().String()+"-"+tableName,
+	)
+	require.NoError(t, err)
+
+	return uploadOutput
+}
+
+func RecordsFromWarehouse(
```

Comment:
Is this function the same as the other helper?

Reply:
Renamed to RetrieveRecordsFromWarehouse with the comment.